services/collection.js

```diff
@@ -166,7 +166,7 @@
                                     that.paginate = false;
 
                                     // Promise
-                                    resolve(that.models);
+                                    resolve(response.data.payload);
```

Comment:
This was changed to the models for the multi-type autocomplete.  Either way, it doesn't really make any sense to return the payload directly when using a collection, as you could easily just use a basic XHR request and avoid the overhead of turning the JSON to instantiated objects.

Reply:
The code is changed from last week and it's may cause to generate the error `Maximun call stack sized` (https://app.asana.com/0/596117517607240/627067968383978). 
I'll change to use XHR but I think it will need much time for me to refactor all the code.